common-tasks/dispvm.md

```diff
@@ -92,8 +137,8 @@ However, it is possible to start an arbitrary application in a DispVM directly f
 Customizing Disposable VMs
 --------------------------
 
-You can change the template used to generate the Disposable VM, and change settings used in the Disposable VM savefile. 
-These changes will be reflected in every new Disposable VM. 
+You can change the template used to generate the Disposable VMs, and change settings used in the Disposable VM savefile. 
+These changes will be reflected in every new Disposable VM spawned from that template. 
```

Comment:
Is "spawned from" different from "launched from" and "based on"? "Spawned from" suggests that "that template" (a DVM Template?) is the parent, and the DispVM is the child, which is usually how we describe the relationship between TemplateVMs and TemplateBasedVMs. So, I'm guessing that "spawned from" is synonymous with "based on," and both are distinct from "launched from."